> ##[error] Manifest: The filename must match the PackageIdentifier and ManifestType properties from the manifest. actual 'AnalogDevices.LTspice.locale.zh-CN.yaml', expected 'AnalogDevices.LTspice.locale.en-US.yaml'

--- a/manifests/a/AnalogDevices/LTspice/24.1.9.1/AnalogDevices.LTspice.locale.zh-CN.yaml
+++ b/manifests/a/AnalogDevices/LTspice/24.1.9.1/AnalogDevices.LTspice.locale.zh-CN.yaml
@@ -3,7 +3,7 @@
 
 PackageIdentifier: AnalogDevices.LTspice
 PackageVersion: 24.1.9.1
-PackageLocale: en-US
+PackageLocale: zh-CN
 # Publisher:
 PublisherUrl: https://www.analog.com/cn/
 PublisherSupportUrl: https://www.analog.com/cn/support.html

